src/libLookup/Lookup.cpp

```diff
@@ -1203,82 +1214,78 @@ bool Lookup::ProcessGetStateDeltaFromSeed(const bytes& message,
+    LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
+              "Messenger::GetLookupGetShardsFromSeed failed.");
+    return false;
+  }
 
-  // Peer requestingNode(from.m_ipAddress, portNo);
-  // bytes msg = {MessageType::LOOKUP,
-  // LookupInstructionType::SETSHARDSFROMSEED};
+  Peer requestingNode(from.m_ipAddress, portNo);
+  bytes msg = {MessageType::LOOKUP, LookupInstructionType::SETSHARDSFROMSEED};
 
-  // lock_guard<mutex> g(m_mediator.m_ds->m_mutexShards);
+  lock_guard<mutex> g(m_mediator.m_ds->m_mutexShards);
 
-  // if (!Messenger::SetLookupSetShardsFromSeed(
-  //         msg, MessageOffset::BODY, m_mediator.m_selfKey,
-  //         SHARDINGSTRUCTURE_VERSION, m_mediator.m_ds->m_shards)) {
-  //   LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
-  //             "Messenger::SetLookupSetShardsFromSeed failed.");
-  //   return false;
-  // }
+  if (!Messenger::SetLookupSetShardsFromSeed(
+          msg, MessageOffset::BODY, m_mediator.m_selfKey,
+          SHARDINGSTRUCTURE_VERSION, m_mediator.m_ds->m_shards)) {
+    LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
+              "Messenger::SetLookupSetShardsFromSeed failed.");
+    return false;
+  }
 
-  // P2PComm::GetInstance().SendMessage(requestingNode, msg);
+  P2PComm::GetInstance().SendMessage(requestingNode, msg);
 
-  // return true;
+  return true;
 }
 
 // Ex-Archival node code
 bool Lookup::ProcessSetShardFromSeed([[gnu::unused]] const bytes& message,
                                      [[gnu::unused]] unsigned int offset,
                                      [[gnu::unused]] const Peer& from) {
-  LOG_GENERAL(WARNING, "Function not in used");
-  return false;
-  // LOG_MARKER();
+  LOG_MARKER();
 
-  // DequeOfShard shards;
-  // PubKey lookupPubKey;
-  // uint32_t shardingStructureVersion = 0;
-  // if (!Messenger::GetLookupSetShardsFromSeed(
-  //         message, offset, lookupPubKey, shardingStructureVersion, shards)) {
-  //   LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
-  //             "Messenger::GetLookupSetShardsFromSeed failed.");
-  //   return false;
-  // }
+  DequeOfShard shards;
+  PubKey lookupPubKey;
+  uint32_t shardingStructureVersion = 0;
+  if (!Messenger::GetLookupSetShardsFromSeed(
+          message, offset, lookupPubKey, shardingStructureVersion, shards)) {
+    LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
+              "Messenger::GetLookupSetShardsFromSeed failed.");
+    return false;
+  }
 
-  // if (shardingStructureVersion != SHARDINGSTRUCTURE_VERSION) {
-  //   LOG_GENERAL(WARNING, "Sharding structure version check failed. Expected:
-  //   "
-  //                            << SHARDINGSTRUCTURE_VERSION
-  //                            << " Actual: " << shardingStructureVersion);
-  //   return false;
-  // }
+  if (shardingStructureVersion != SHARDINGSTRUCTURE_VERSION) {
+    LOG_GENERAL(WARNING, "Sharding structure version check failed. Expected:"
```

Comment:
Can use this instead:
`LOG_CHECK_FAIL("Sharding structure version", shardingStructureVersion, SHARDINGSTRUCTURE_VERSION);`